proxy/src/schema/registry.rs

```diff
@@ -91,6 +102,7 @@ impl Registry {
         author: &ed25519::Pair,
         domain: String,
         name: String,
+        maybe_pid: Option<String>,
```

Comment:
Why is this an `Option<String>` and not the actual type librad uses for project ids?

Reply:
Because I couldn't figure out how to convert it to a format Serde can deserialize :grimacing:

Reply:
No wait, that's not true. It's because librad doesn't re-export git2 so I couldn't construct an Oid from a string without adding git2 as a dependency, so I used a string instead and... then forgot to go back and change it. I'm working on a fix now.

Reply:
Hmm upon further review it appears I just got very confused and then forgot about the issue when I opted to use a string as an intermediary placeholder. I'm still pretty confused so maybe we can talk about this on Monday?

Reply:
Addressed in a608b52 to use proper id types.